executor/executor_test.go

```diff
@@ -1850,6 +1850,12 @@ func (s *testSuiteP1) TestGeneratedColumnRead(c *C) {
 	result = tk.MustQuery(`SELECT * FROM test_gc_read_m`)
 	result.Check(testkit.Rows(`1 2 4`, `2 3 6`))
 
+	// Test for issue #12233
+	tk.MustExec(`create table testjson( id int auto_increment not null primary key, j json )default charset=utf8 engine=innodb;`)
```

Comment:
Could you simplify this table schema?

Reply:
Thank you. I had simplified the table schema.